Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Cortex_Endpoint_Enrichment.yml

```
@@ -0,0 +1,1431 @@
+                    right:
+                      value:
+                        simple: Cortex Core - IR
+                - - left:
+                      iscontext: true
+                      value:
+                        simple: modules.state
+                    operator: isEqualString
+                    right:
+                      value:
+                        simple: active
+                root: modules
+          operator: isExists
+      - - left:
+            iscontext: true
+            value:
+              complex:
+                root: inputs.RemoteIP
+          operator: isExists
+      label: "yes"
+    continueonerrortype: ""
+    id: "9"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "8"
+      "yes":
+      - "6"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Checks if there is an active instance of the Core - IR integration enabled and input values are defined to pull enrichment data.
+      id: 17e91dd6-4328-46db-85de-8fd0b7e50f84
+      iscommand: false
+      name: Is Core IR enabled and input value defined?
+      type: condition
+      version: -1
+    taskid: 17e91dd6-4328-46db-85de-8fd0b7e50f84
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 460,
+          "y": 100
+        }
+      }
+  "10":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              complex:
+                accessor: endpoint_id
+                root: Core.Endpoint
+          operator: isExists
+          right:
+            value: {}
+      label: "yes"
+    continueonerrortype: ""
+    id: "10"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "8"
+      "yes":
+      - "26"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Check whether the last command returned endpoint information or not.
+      id: 1616c292-8f7d-4c62-892f-352b62c56290
+      iscommand: false
+      name: Was there a response?
+      type: condition
+      version: -1
+    taskid: 1616c292-8f7d-4c62-892f-352b62c56290
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 670,
+          "y": 660
+        }
+      }
+  "12":
+    continueonerrortype: ""
+    id: "12"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "21"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      gridfield:
+        simple: asmsystemids
+      keys:
+        simple: type,id,link
+      val1:
+        simple: ASSET-TYPE
+      val2:
+        simple: CORTEX-Endpoint
```

```suggestion
        simple: Cortex Endpoint
```

I think this would be consistency with other `ASSET-TYPE` set for other enrichment playbooks and `source` fields in this playbook. Overall I think the name "Cortex Endpoint" makes sense and encompasses XDR and XSIAM.